apps/astarte_housekeeping/lib/astarte_housekeeping/local.ex

```diff
@@ -0,0 +1,46 @@
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Housekeeping.Local do
+  use TypedEctoSchema
+
+  alias Astarte.Housekeeping.UUID
+  alias Astarte.Housekeeping.INet
+
+  @primary_key false
+  @schema_prefix "system"
+  schema "local" do
```

Comment:
```suggestion
  typed_schema "local" do
```

watch out: `TypedEctoSchea`'s macro is `typed_schema`, `schema` is the one from `Ecto.Schema`